examples/styles.rs

```diff
@@ -1,14 +1,17 @@
 // extern crate env_logger;
 extern crate gfx;
+extern crate gfx_device_gl;
 extern crate gfx_window_glutin;
 extern crate glutin;
 extern crate gfx_text;
 
-use gfx::traits::Stream;
+use gfx::traits::Device;
 use gfx_window_glutin as gfxw;
 use gfx_text::{HorizontalAnchor, VerticalAnchor};
 use glutin::{WindowBuilder, Event, VirtualKeyCode, GL_CORE};
 
+type Encoder = gfx::Encoder<gfx_device_gl::Resources, gfx_device_gl::CommandBuffer>;
```

Comment:
You don't need this define here. When specifying the type, you can just use `gfx_Encoder<_, _>` instead.
This would allow getting rid of the `gfx_device_gl` direct dependency from here.